app/views/layouts/external.html.erb

```diff
@@ -1,11 +1,21 @@
 <% content_for?(:page_title) ? yield(:page_title) : fallback_title %>
 
```

Comment:
I think we should try to keep it so that the external layout has no current user info.  I wonder if we need to reconsider the base controller for the errors controller.